src/SFA.DAS.EmployerAccounts/Data/UserRepository.cs

```diff
@@ -127,5 +128,33 @@ public async Task<Users> GetAllUsers()
                 UserList = result.ToList()
             };
         }
+
+        public async Task<IEnumerable<DateTime>> GetAornPayeQueryAttempts(Guid userRef)
+        {
+            var parameters = new DynamicParameters();
+            parameters.Add("@UserRef", userRef, DbType.Guid);
+
+            var query = await _db.Value.Database.Connection.QueryAsync<DateTime>(
+                sql: "[employer_account].[GetUserAornAttempts]",
+                param: parameters,
+                transaction: _db.Value.Database.CurrentTransaction.UnderlyingTransaction,
+                commandType: CommandType.StoredProcedure);
+
+            return query.ToList();
```

Comment:
Won't the query just return a single result?

Reply:
No it returns a list of the attempts.

Reply:
Makes sense given the name of the method.